lib/index.js

```diff
@@ -78,8 +73,12 @@ export default {
         const fileText = TextEditor.buffer.cachedText;
 
         // Is flow enabled for current file ?
-        if (fileText.indexOf('@flow') === -1 && !this.enableAll) {
-          return [];
+        const firstComStart = fileText.indexOf('\/*');
```

Comment:
The old test would detect @flow in strings etc.
This is slightly stricter.

`/* @flow */` isn't a very good test as you can put this for example:
```
/*
@flow
something else 
*/
```